packages/ag-charts-community/src/chart/data/dataController.ts

```diff
@@ -202,6 +202,7 @@ export class DataController {
                 dataModel,
                 processedData: {
                     ...processedData,
+                    // @ts-ignore
```

Comment:
Let's make sure to clean this out before final merge.

Reply:
Nope - will fix in PR 3